agent/browser/ie/wptbho/wpt.cc

```diff
@@ -935,6 +940,29 @@ CComPtr<IHTMLElement> Wpt::FindDomElementInDocument(CString tag,
 }
 
 /*-----------------------------------------------------------------------------
+	Retrieve load event end time
+-----------------------------------------------------------------------------*/
+LONGLONG Wpt::GetLoadEventEnd() {
+  AtlTrace(_T("[wptbho] - Wpt::GetLoadEventEnd()"));
+
+  if (Exec(LOAD_EVENT_END_FUNCTION)) {
+    _variant_t timing;
+    
+    if (Invoke(GET_LOAD_EVENT_END_TIMING, timing)) {

```

Comment:
As per the specification of the performance timing API:

```
loadEventEnd attribute

This attribute must return the time when the load event of the current document is completed. It must return zero when the load event is not fired or is not completed.
```

This function cannot return `null`.